src/data_gradients/feature_extractors/object_detection/bounding_boxes_area.py

```diff
@@ -75,6 +88,59 @@ def aggregate(self) -> Feature:
         )
         return feature
 
+    @staticmethod
+    def _compute_histogram(df: pd.DataFrame, transform_name: str, min_bin_val: int = 1) -> dict:
```

Comment:
It may not be the only place, but we renamed the json keys to 
```
                    'bin_width': 1,      # width between histogram bins
                    'min_value': 1,      # min size value in the histogram
                    'max_value': 4,      # max size value in the histogram
```

So it would be great if we can also change the variables names to match. Here `min_bin_val` -> `min_value`, but there might be other variables to also update